runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/DataflowRunnerTest.java

```diff
@@ -256,6 +256,22 @@ private Pipeline buildDataflowPipeline(DataflowPipelineOptions options) {
     return p;
   }
 
+  private Pipeline buildDataflowPipelineWithLargeGraph(DataflowPipelineOptions options) {
```

Comment:
minor: this method and the previous one (buildDataflowPipeline) could be static

Reply:
Done.